src/compiler.js

```diff
@@ -220,7 +220,17 @@ class Compiler {
         throw e
       }
 
+      // Look for `node_modules` in the same dir as `subgraph.yaml`.
       let libs = path.join(baseDir, 'node_modules')
+      if (!fs.existsSync(libs)) {
+        // It's not there, so look in up to three directories above.
+        for (let i = 1; i <= 3; i++) {
```

Comment:
I'd check _all_ parent directories. I know `/` doesn't make sense to check but directories are rarely deeper than, say 10 levels, so this check would be cheap to run for all of them, and special-casing to 3 feels unnecessary.

Reply:
I took the lazy/simple route and bumped 3 to 100, so we don't have to use a complicated logic that makes it hard to tell if it can loop forever.